### ISSUE TITLE
Alpine 3.15-based images?

### ISSUE THREAD

**Issue by p-bakker** [adoptium/containers#135]
Looks like the Alpine-based images are on Alpine 3.14.

**Comment by gdams** [comment]
@tianon what's the upstream strategy here? Are we okay to bump our Dockerfiles to depend on 3.15 and continue using the same tag or do we need to support 3.14 in parallel?

**Comment by tianon** [comment]
It's mostly up to you -- a common strategy is to introduce `alpineX.Y` tags such that you can support both concurrently (or even just so that when you update, users who can't update yet can continue using the old now-unsupported tag with 3.14 easily without having to dig out a digest), but if you think the breakage is minimal and most users should be fine to "just update" then you can just update it in-place.

See https://hub.docker.com/_/golang for a reasonably straightforward example of what I mean (there we support at most two versions of Alpine since we have to build from source so there's a non-trivial overhead resource cost for each supported version combination).

**Comment by aeimer** [comment]
@tianon @gdams actually alpine 3.15 breaks our builds because of rootfs and kaniko. It would be nice to have -alpine and -alpine3.14 / -alpine3.15 tags in future!
